stylesheets/style.css

```diff
@@ -0,0 +1,367 @@
+    margin-bottom: 0px;
+}
+
+.page-3 .subnav {
+    float: initial;
+    display: inline;
+}
+
+.subnav {
+    overflow: hidden;
+    margin-bottom: 10px;
+}
+
+.subnav-content {
+    display: none;
+    position: relative;
+    left: 0;
+    background-color: inherit;
+    width: 100%;
+    z-index: 1;
+    text-align: inherit;
+    font-size: medium;
+}
+
+.page-2 .subnav-content {
+    position: absolute;
+    background-color: lightpink;
+    width: 16%;
+    margin-left: 49%;
+    margin-right: 35%;
+}
+
+.page-2 .subnav-content a { 
+    margin: 1vh;
+    color: darkblue;
+}
+
+.subnav-content a {
+    float: right;
+    color: whitesmoke;
+    text-decoration: none;
+}
+
+.subnav-content a::before {
+    content: "✯ ";
+}
+
+.subnav:hover .subnav-content {
+    display: block;
+    grid-column: 10 / 13;
+    grid-row: 5 / 12;
+}
+
+.subnav-content a:hover {
+    background-color: #eee;
+    color: black;
+}
+
+li.page-1 {
+    text-shadow: 1.8px 1.8px 1px lightslategray;
+    text-align: right;
+    list-style-type: none;
+    padding-top: 10px;
+    font-size: 4vh;
+}
+
+li.page-1:not(:last-child) { 
+    margin-bottom: 4vh;  
+}
+
+li.page-1:any-link {
+    color: #2C04BF;
+}
+
+nav :any-link:hover {
+    background-color: black;
+    color: #EB39C4;
+    font-weight: bold;
+}
+
+div.alien {
+    grid-area: 5 / 1 / 12 / 4;
+    order: 0;
+    margin: auto;
+}
+
+img.alien {
+    max-width: 90%;
+}
+
+div.statement {
+    grid-area: 5 / 5 / 12 / 9;
+    order: 1;
+}
+
+div p.page-1 {
+    font-family: 'Imprint', 'Times New Roman', Impact, serif, monospace, sans-serif;
+    font-size: 7vh;
+    font-weight: bold;
+    text-align: center;
+    text-shadow: 1px 1px rgba(255,255,255,0.8);
+}
+
+footer.page-1 {
+    grid-column: 1 / 13;
+    grid-row: 12 / span 1;
+    order: 3;
+    vertical-align: center;
+}
+
+footer p.page-1 {
+    font-family: fantasy;
+    font-style: oblique;
+    font-size: 2.5vh;
+    text-align: center;
+    color: orangered;
+}
+
+nav.page-2 {
+    font-size: 2.2vh;
+    background-color: lightpink;
+    text-transform: uppercase;
+    list-style-type: none;
+    text-align: center;
+    padding: 2px;
+}
+
+nav.page-2 li {
+    display: inline
+}
+
+nav.page-2 li:not(:last-child) { 
+    margin-right: 6vh;  
+}
+
+.disclaimer {
+    font-size: 3.5vh;
+    margin: 4vh 4vh;
+    align-items: center;
+    display: inline;
+    text-align: center;
+}
+
+.disclaimer img {
+    max-height: 3.5vh;
+    padding-inline: 3px;
+    vertical-align: text-bottom;
+    margin-top: 4vh;
+}
+
+.facts {
+    padding: 2vh
+}
+
+.facts li {
+    font-size: 2vh;
+    margin-bottom: .7vh;
+}
+
+#dancers {
+    align-items: center;
+    margin: auto;
+}
+
+footer.page-2 p{
+    font-family: sans-serif;
+    text-align: center;
+    color: #135EF2;
+}
+
+header.page-3 {
+    margin: 0 30px 0 10px;
+}
+
+footer.page-3 {
+    width: 100%;
+    text-align: center;
+}
+
+nav.page-3 {
+    display: flex;
+    -ms-display: flex;
```

Comment:
Was this maybe necessary for microsoft browsers in the past? I shouldn't think it would be needed anymore. Lots of new features are added to CSS as vendor-specific extensions, but once they are widely adopted, it's nice to be able to drop the vendor versions.